tailscale-file: add page (#7844)
tldr-pages/tldr: pages/common/tailscale-file.md

# tailscale file

> Send files across connected devices on a Tailscale network.
> It currently does not support sending files to devices owned by other users even on the same Tailscale network.

> > More information: <https://tailscale.com/kb/1106/taildrop/>

Author: ```suggestion
> More information: <https://tailscale.com/kb/1106/taildrop/>.
```

> - Send a file to another device on the Tailscale network:

Reviewer: Based on the `tailscale-up` page:
```suggestion
- Send a file to a specific node:
```

> `sudo tailscale file cp {{path/to/file}} {{hostname_or_ip}}:`

Reviewer 2: ```suggestion
`sudo tailscale file cp {{path/to/file}} {{hostname|ip}}:`
```

I suggest always replace all `or` with `|` because it makes unambiguous multi word alternatives.

> `sudo tailscale file cp {{/path/to/file}} {{hostname_or_ip}}:`

Reviewer: ```suggestion
`sudo tailscale file cp {{path/to/file}} {{hostname_or_ip}}:`
```

> - Receive a file that was sent to the device:

Reviewer: The file was already received, the sender could be disconnected from the network. This is move the file from a temporal directory to a specific one.
```suggestion
- Store files that was sent to the current node into a specific directory:
```

> - Store files that was sent to the current node into a specific directory:

Reviewer: my typo :joy: 
```suggestion
- Store files that were sent to the current node into a specific directory:
```
